packages/devops/configs/servers.conf

```diff
@@ -258,7 +258,7 @@ server {
   include /etc/nginx/h5bp/basic.conf;
 
   location / {
-              proxy_pass http://localhost:8040;
+              proxy_pass http://localhost:8070;
```

Comment:
I spent the entire day scratching my head on why admin-panel of the deployment doesn't work, and any code changes just didn't seem to have any effects... turned out it was because of this wrong port 🤕